### PR TITLE
Update file1

### PR DESCRIPTION
**Pull request by prashanthkalasani**
image

### CODE REVIEW

**Comment by prashanthkalasani** [comment]
hai

**Comment by Madhava-Chary-007** [comment]
hai bro

File: file1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-prashanth
```
Comment (Madhava-Chary-007) [inline file1:1]: hi

File: file1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
-sai
-uday
-sunil
```
Comment (Madhava-Chary-007) [inline file1:4]: sunil name is mistake

Comment (prashanthkalasani) [reply file1:4]: ok i checked is correct

**Pull request merged**